update.php

```diff
@@ -567,6 +567,7 @@ function update_1069() {
 	q("ALTER TABLE `fcontact` ADD `request` CHAR( 255 ) NOT NULL AFTER `photo` ");
 }
 
+/// @TODO Still meeded?
```

Comment:
I'm not sure the TODO tag is the best option to discuss code changes. I would expect a TODO to outline a future change, not to ask a question.